spec/generators/sorbet_test_cases.rb

```diff
@@ -305,6 +305,17 @@
 T.assert_type!(Wizard::House::Ravenclaw, Wizard::House)
 T.assert_type!(Wizard::House::Slytherin, Wizard::House)
 
+# Serialization
+T.assert_type!(
+  wizard.owl_results,
+  T.any(T::Array[T.untyped], T::Boolean, Float, T::Hash[T.untyped, T.untyped], Integer, String)
+)
+T.assert_type!(
+  wizard.patronus_characteristics,
+  T.any(T::Array[T.untyped], T::Boolean, Float, T::Hash[String, T.untyped], Integer, String)
+)
+T.assert_type!(wizard.pets, T::Array[T.untyped])
```

Comment:
In the generated type, pets can be a nilable array. Should we assert a nilable type here?
```
  sig { returns(T.nilable(T::Array[T.untyped])) }
  def pets
```

Reply:
Fixed :)